rollup.config.js

```diff
@@ -34,9 +32,10 @@ export default {
 			extensions: ['.ts'],
 			exclude: 'node_modules/**',
 			babelHelpers: 'bundled'
-		})
+		}),
+		visualizer()
```

Comment:
will this always create visualisation as part of the build? Ideally, we should do this only when a developer wants to do that.

Reply:
With current setting, it will create a file with name stats.html after the build. The file will not open automatically. Will check if we have any option to disable it or make it explicit.

Reply:
Fixed this. Added a separate command to generate build analysis data.